Cannot find module '../../bower.json'

The issue came up when installing the addon.

There's an error at https://github.com/acoustep/ember-cli-foundation-6-sass/blob/master/blueprints/ember-cli-foundation-6-sass/index.js#L3.

```
Cannot find module '../../bower.json'
Error: Cannot find module '../../bower.json'
    at Function.Module._resolveFilename (module.js:438:15)
    at Function.Module._load (module.js:386:25)
    at Module.require (module.js:466:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/dev/ember-app/node_modules/ember-cli-foundation-6-sass/blueprints/ember-cli-foundation-6-sass/index.js:3:19)
    at Module._compile (module.js:541:32)
    at Object.Module._extensions..js (module.js:550:10)
    at Module.load (module.js:456:32)
    at tryModuleLoad (module.js:415:12)
    at Function.Module._load (module.js:407:3)
```

I think the reason is that `require` is looking up relative to the CWD instead of file's directory.

--- a/blueprints/ember-cli-foundation-6-sass/index.js
+++ b/blueprints/ember-cli-foundation-6-sass/index.js
@@ -1,6 +1,8 @@
+/* jshint node:true */
+
 var fs          = require('fs');
 var path        = require('path');
-var bower       = require('../../bower.json');
+var bower       = require(path.resolve(__dirname, '../../bower.json'));
 
 module.exports = {
   description: 'install ember-cli-foundation-6-sass into a typical project',

